entrypoints/content/pages/profile.tsx

```diff
@@ -0,0 +1,128 @@
+// Components:
+import { Avatar, AvatarFallback, AvatarImage } from '../components/ui/avatar'
+import { Button } from '../components/ui/button'
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuLabel,
+  DropdownMenuSeparator,
+  DropdownMenuTrigger,
+} from '../components/ui/dropdown-menu'
+import { Separator } from '../components/ui/separator'
+import Comment from '../components/secondary/comment/Comment'
+
+// Imports:
+import { Camera } from 'lucide-react'
+import { Link2Icon } from '@radix-ui/react-icons'
+
+// Typescript:
+import { TOPICS } from 'constants/database/comments-and-replies'
+import { FieldValue } from 'firebase-admin/firestore'
+
+// Functions:
+function Profile({ children }: { children: React.ReactNode }) {
+  // Constants:
+  const SAMPLE_USER_INFO = {
+    fullName: 'Ben Holmes',
+    username: '@BenHolmesDev',
+  }
+
+  const SAMPLE_CONTENT = `Right when I heard Stephen first talk I had a gut feeling that it was him. 
+
+Just the way he talked about the situation it's like he had already come to terms with something as crazy and as random as an explosion. 
+
+
+Bro had a eulogy ready for her and everything.`
+
+  const SAMPLE_COMMENT = {
+    id: 'some uuid',
+    URLHash: 'string',
+    domain: 'sdsd',
+    URL: 'https://www.example.co.uk:443/blog/article/search?docid=720&hl=en',
+    author: 'dfsdfs',
+    replyCount: 0,
+    sentiment: 5,
+    topics: [TOPICS.ANTHROPOLOGY],
+    // COULDN'T UNDERSTAND WHAT FIELD VALUE IS SO KINDA CIRCUMVENTING THIS
+    // SHOULD NOT TO BE A PROBLEM WHEN WE INTEGRATE FIRESTORE
+    createdAt: new Date(Date.now() - 17 * 60 * 60 * 1000) as unknown as FieldValue,
+    creationActivityID: '',
+    body: SAMPLE_CONTENT,
+    voteCount: {
+      up: 40,
+      down: 4,
+      controversy: 5,
+      wilsonScore: 5,
+    },
+    hateSpeech: {
+      isHateSpeech: false,
+    },
+  }
+
+  // Return:
+  return (
+    <main className='w-full pt-16 bg-white' style={{ height: 'calc(100% - 68px)' }}>
+      <div className='w-full h-fit p-8 pb-6 flex flex-row gap-10'>
+        <div className='h-fit relative rounded-full brightness-100 cursor-pointer transition-all duration-500'>
+          <Avatar className='w-32 h-32 brightness-75'>
+            <AvatarImage src={'https://github.com/shadcn.png'} />
```

Comment:
minor comment and probably can be fixed by eslint or prettier

`src='https://github.com/shadcn.png'` also works fine, IMO the curly braces here (and in other places) seems redundant and ought to be removed (from here and other places)